web/autograder_service.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/autograde/aguis/database"
 	scms "github.com/autograde/aguis/scm"
 	"github.com/autograde/aguis/web/auth"
+	"github.com/jinzhu/gorm"
```

Comment:
Here we are leaking internal error types from Gorm. I've looked at the upspin `errors` package. I propose we adopt something like that, creating our own error types that can be checked. Let's do it stepwise, by adding this one error type first.

Reply:
Fixed it by adding a new simple error to already existing set of custom errors for `web` package, because our goal here is just to avoid unnecessary logging.